README.md

```diff
@@ -29,38 +29,49 @@ it).
 
 #### Get repo
 
-.. and change into the `src` directory
+.. and create a out-of-tree `build` directory
 
 ```bash
 git clone https://github.com/hzeller/timg.git
-cd timg/src
+mkdir build
+cd build
 ```
 
 #### Compiling on Debian/Ubuntu
 ```bash
-sudo apt-get install libwebp-dev libgraphicsmagick++-dev    # required libs.
+sudo apt install cmake git g++ pkg-config libgraphicsmagick++-dev    # required libs.
 
 # If you want to include video decoding, also install these additional libraries
-sudo apt-get install pkg-config libavcodec-dev libavformat-dev libswscale-dev
+sudo apt install libavcodec-dev libavformat-dev libswscale-dev
 
-make WITH_VIDEO_DECODING=1   # or 0, if you don't want video decoding
+# If you want to recreate the man page
+sudo apt install pandoc
+
+cmake ..
```

Comment:
Maybe we can make this similar in both install descriptions with one cmake line but that already contains the parameter (like below in the :apple:  installation)

```bash
cmake ../ -DWITH_VIDEO_DECODING=On  # of Off to for no video
```

Here we use `On` even though not needed, but we make it simple to discover that there is this setting. This way, we consume less attention span for someone not having to read the following two lines of comments :)